server/lib/queries.js

```diff
@@ -1049,6 +1049,24 @@ const getBalances = async (collectiveIds, until = new Date()) =>
     { type: sequelize.QueryTypes.SELECT, replacements: { ids: collectiveIds, until } },
   );
 
+const getBalancesInHostCurrency = async (collectiveIds, until = new Date()) =>
+  sequelize.query(
+    `
+      SELECT
+        t."CollectiveId",
+        COALESCE(SUM(ROUND(t."netAmountInCollectiveCurrency" * t."hostCurrencyFxRate")), 0) AS "balance"
```

Comment:
I just checked and we have some transactions with a null `hostCurrencyFxRate`:

```sql
SELECT * FROM "Transactions" t WHERE "hostCurrencyFxRate" IS NULL AND "deletedAt" IS NULL
```

We probably want to fix these entries and add a non-null constraint on `hostCurrencyFxRate`, but if we're not ready to do it now we could do:
```suggestion
        COALESCE(SUM(ROUND(t."netAmountInCollectiveCurrency" * COALESCE(t."hostCurrencyFxRate", 1))), 0) AS "balance"
```

Reply:
Can we run a migration and default hostCurrencyFxRate to 1?

Reply:
I fixed the recent existing records manually, they were created in the invoice worker.
Also, I added set default value of hostCurrencyFxRate to 1.